scripts/validation/Invoke-ForgeNonProductionPersistentRuntimeRollbackProof.ps1

```
@@ -0,0 +1,430 @@
+    Copy-Item -LiteralPath $forgeBinary -Destination $artifactPath
+    $forgeSha256 = (
+        Get-FileHash -Algorithm SHA256 -LiteralPath $artifactPath
+    ).Hash.ToLowerInvariant()
+
+    $sovereignSource = Join-Path $proofRoot 'sovereign-source'
+    Copy-Item -Recurse -LiteralPath (
+        Join-Path $sovereignRepository 'packages\terrabuild\kernels\terraforge.kernel.valuation'
+    ) -Destination $sovereignSource
+    $sovereignManifest = Join-Path $sovereignSource 'Cargo.toml'
+    $env:CARGO_TARGET_DIR = $sovereignTarget
+    Invoke-Checked cargo build --release --offline --manifest-path $sovereignManifest
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-Checked' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)